src/test/java/org/simlar/simlarserver/services/pushnotification/apple/ApplePushNotificationServiceTest.java

```diff
@@ -66,7 +66,7 @@ public void verifyConfiguration() {
     public void testConnectToAppleWithoutCertificateForbidden() {
         try {
             new RestTemplateBuilder()
-                    .requestFactory(OkHttp3ClientHttpRequestFactory::new)
+                    .requestFactory(() -> new OkHttp3ClientHttpRequestFactory())
```

Comment:
Or, if we don't need to pass any parameters:
`                    .requestFactory(OkHttp3ClientHttpRequestFactory.class)`